src/ser/mod.rs

```diff
@@ -16,6 +16,16 @@ where
     value.serialize(&mut s)
 }
 
+/// Serializes `value` into `writer` in a pretty way.
+pub fn to_writer_pretty<W, T>(writer: W, value: &T, config: PrettyConfig) -> Result<()>
```

Comment:
I think it should be called differently, since it doesn't return a writer. Maybe `write_pretty`?

Reply:
I guess it could be, but I was just copying the name from `to_writer` and then adding `_pretty`, in the same way as `to_string` and `to_string_pretty`. If you think about it in terms of the full path, it's `ron::ser::to_writer_pretty`, which does actually make sense.

Reply:
oh, sorry, `to_writer` is also a bad name, uh